### DISCUSSION TITLE
Is there a golang based client example to support TLS communication?

### DISCUSSION THREAD

**Discussion by mvkzzz** [memcached/memcached#1084] [Q&A]
Since TLS  is supported from version 1.5.13, I want to build a memcached client based on golang and communicate with memcached server over TLS, what should I do?

**Comment by dormando** [comment]
It looks like the `gomemcache` client supports TLS already: https://github.com/bradfitz/gomemcache/commit/24af94b0387418c51cc45a2e1fe6d4d1bef8a0fd

sorry for the long delay in responding.

**Comment by mvkzzz** [comment]
Thanks for your kind help



发自我的iPhone


------------------ Original ------------------
From: dormando ***@***.***&gt;
Date: Sat,Oct 28,2023 1:23 AM
To: memcached/memcached ***@***.***&gt;
Cc: mvkzzz ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [memcached/memcached] Is there a golang based client example tosupport TLS communication? (Discussion #1084)





 
It looks like the gomemcache client supports TLS already: ***@***.***
 
sorry for the long delay in responding.
